sdk/storage/azure-storage-queue/src/main/java/module-info.java

```diff
@@ -2,21 +2,16 @@
 // Licensed under the MIT License.
 
 module com.azure.storage.queue {
+    requires transitive com.azure.xml;
```

Comment:
Do we need to make this transitive? Do we expect customers to use any of the Xml serializer APIs from azure-xml directly?